wandb/sdk_py27/wandb_artifacts.py

```diff
@@ -167,6 +168,9 @@ def __init__(
         self._distributed_id = None
         self._logged_artifact = None
         self._incremental = False
+        self._client_id = util.generate_id(64)
```

Comment:
Have we done the collision math on this? It might be safer to keep this high, at `256` which is the max the backend supports.

Reply:
Yeah, so there are 36 character options, and with a length of 64, that is 36**64=4011991914547630480065053387702443812690402487741812225955731622655455723258857248542161222254985216 options. An astronomical number. Running `DOMAIN=36**128; COUNT=int(20e6); 1 - math.prod([1 - (i/DOMAIN) for i in range(COUNT)])` in python literally just grounds out to 0. I can increase it, still, maybe 128? Honestly it is so big lol